examples/vpc_service_control/vpc_sc_project/variables.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "project_id" {}
+
+variable "service_account_file" {}
+
+variable "organization_id" {}
+
+variable "billing_account_id" {}
+
+variable "cloud_router_region" {}
+
+variable "ip_addr_of_onprem_vpn_router" {}
+
+variable "shared_secret_string_for_vpn_connection" {}
+
+// you can get your access policy name (which is actually a number) by running:
+// `gcloud organizations list` <- to get your org number
+// `gcloud access-context-manager policies list --organization=<your org number>`   <- to get the access policy name
+variable "access_policy_name" {}
```

Comment:
This assumes the policy is already created.
Consider using the [org policy module](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/blob/master) the same way the other examples use it. See [simple_example](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/blob/master/examples/simple_example/main.tf)

Reply:
Done.

Reply:
I think what you did is remove the comments but you didn't edit main.tf to use the org policy module in main.tf
Also, remember to add descriptions to all variables.

Reply:
This change is still pending.
I think you may have used VPC Service Controls with the GUI once with another project and the default access policy was created as mentioned [here](https://cloud.google.com/access-context-manager/docs/create-access-policy)
You should either add a step to use a gcloud command to create the policy or use the org policy module in this repo.

Reply:
Updated the README docs with the instructions on how to use an existing access policy, or to create one if it doesn't exist.